### PR TITLE
Updated references to Medic Mobile

### PR DESCRIPTION
**Pull request by abbyad**

### CODE REVIEW

**Comment by abbyad** [comment]
Apologies for the spammy review request, this PR covered a lot of code owners! Only one reviewer needed here, @mrjones-plip can you review this?

**Comment by mrjones-plip** [comment]
Yes, will grab today!

File: content/en/apps/guides/messaging/gateway/phones.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ relatedContent: >
   apps/guides/messaging/gateway/configuration
 ---
 
-In order to implement the Medic Mobile Standard package at your site, you will require an Android device for the Medic gateway. You may use an existing Android device and are not required to purchase a new one. However, please note that the Android device must always stay in your organization’s office or facility, and must be connected to stable and continuous internet at all times.
```
Comment (mrjones-plip) [inline content/en/apps/guides/messaging/gateway/phones.md:12]: much improved! nice work.

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
I think you got them all but one! See `content/en/core/overview/translations.md`:

```
➜  cht-docs git:(443-rename-medic-mobile) grep -r 'Medic Mobile' content/en               
content/en/core/overview/translations.md:If you don't have an API token, please contact a Medic Mobile developer, product manager, or CTO. Please do not disclose this API token to anyone else.
```

**Comment by mrjones-plip** [comment]
Oh weird, that `translations.md` change I requested, is [actually covered here](https://github.com/medic/cht-docs/pull/444/files#diff-79fc16fa5b66994daa2f1c0518e4af70d6b3fdb4c3b27eec2b24c591c73048c5L64), but I'm still seeing it on disk in the branch?!  I'll approve this PR and we can see if it's still there when we're all back on `master`.  sorry for the confusion!

**Review by mrjones-plip — APPROVED** [review]
approved as long as we check the [one on translations](https://docs.communityhealthtoolkit.org/core/overview/translations/#exporting-changes-from-poeditor-to-github) after it's live!

**Comment by abbyad** [comment]
Confirming that the one one translations is all good.

**Comment by mrjones-plip** [comment]
@abbyad  is good or not good?  You want it to still say "Medic Mobile"?

> please contact a Medic Mobile developer, product manager, or CTO.

The [line in the PR](https://github.com/medic/cht-docs/pull/444/files#diff-79fc16fa5b66994daa2f1c0518e4af70d6b3fdb4c3b27eec2b24c591c73048c5L64) was there to change this line to simply:

> please contact a Medic developer.

but for some reason the first text above is [what's live](https://docs.communityhealthtoolkit.org/core/overview/translations/#exporting-changes-from-poeditor-to-github) (and [in master](https://github.com/medic/cht-docs/blob/master/content/en/core/overview/translations.md#adding-new-keys), obvs.), not the second (still baffled why!?).

**Comment by abbyad** [comment]
Ah, it was in two places 🤦

**Comment by abbyad** [comment]
Thanks for catching that. I have fixed it in https://github.com/medic/cht-docs/pull/445

**Pull request merged**